src/YUILoader.cc

```diff
@@ -156,6 +157,34 @@ void YUILoader::loadUI( bool withThreads )
     }
 }
 
+void YUILoader::loadTestFrameworkPlugin( )
+{
+        if( !getenv("YUI_HTTP_PORT") ) return;
```

Comment:
I'm not sure if we should test it here or delegate the check to the plugin. I mean we duplicate checking the `YUI_HTTP_PORT` value here which is also done in the plugin.

And also the name itself should be shared somehow as a `#define` or a class constant to have one place for changing..

Reply:
So I have renamed it, as we might need to differentiate behavior for testing scenarios and not using variable which is active used to define port number. So I guess we might introduce some default port then.